starter-code/index.html

```diff
@@ -3,40 +3,115 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,700" rel="stylesheet"> 
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<main>
+
+<nav>
+  <img src ="images/spotify-logo.png" alt="Welcome To Spotify">
+  <ul>
+    <li href="#">Premium</li>
+    <li href="#">Discover</li>
+    <li href="#">Help</li>
+    <li href="#">Download</li>
+  </ul>
+</nav>
+
+<section class="first-section">
+    <h1>Music for everyone.</h1>
+    <p>Spotify is now free on mobile, tablet and computer.<br>
+      Listen to the right music, wherever you are.</p>
+</section>
+
+<section class="second-section">
+
+    <h3 class="whatson">What’s on Spotify?</h3>
+
+    <div class ="container-articles">
+    <article class="first-artic pro">
```

Comment:
There is many way to do this, but the one I would choose is, as the layout of the three elements is the same, you can use a list, and add some flex property (it' will be easier to deal with the responsiveness), like this : 

```html 

<ul>
        <li>
          <img src="images/music-icon.png">
          <h3>Millions of Songs</h3>
          <p> There are millions of
            <br>songs on Spotify</p>
        </li>
        <li>
          <img src="images/high-quality-icon.png">
          <h3>HD Music</h3>
          <p>Listen to music as if you
            <br> were listening live</p>
        </li>
        <li>
          <img src="images/devices-icon.png">
          <h3>Stream Everywhere</h3>
          <p>Stream music on your
            <br>smartphone, tablet or
            <br> computer
          </p>
        </li>
      </ul>
```